spyder/__init__.py

```diff
@@ -48,7 +48,12 @@
 DATAPATH = LOCALEPATH = DOCPATH = MATHJAXPATH = JQUERYPATH = ''
 
 
-import os
+import os, sys
+
+# Fixes launching issues with Big Sur (GH14222)
+if sys.platform == "darwin":
+    os.environ['QT_MAC_WANTS_LAYER'] = '1'
```

Comment:
Could you move this to `spyder/app/start.py`? We're already setting other environment variables there to fix other issues.

Reply:
Ok, I am just not sure if this needs to be set before Qt is imported, but I'll try

Reply:
It works